iocage_lib/ioc_fetch.py

```diff
@@ -300,12 +301,22 @@ def fetch_http_release(self, eol, _list=False):
         """
 
         if self.hardened:
-            if self.server == "download.freebsd.org":
+            if self.server is None:
                 self.server = "http://jenkins.hardenedbsd.org"
                 rdir = "builds"
 
-        if self.root_dir is None:
-            self.root_dir = f"ftp/releases/{self.arch}"
+        md = re.match("([0-9]+)[.]([0-9]+)-RELEASE", self.release)
+        if md and (int(md[1]) < 11 or (int(md[1]) == 11 and int(md[2]) < 2)):
```

Comment:
This will require maintenance in perpetuity.  
Instead of hard-coding 11.2, maybe consider something more dynamic as in function `ioc_common.parse_latest_release()` that gets the latest release.

Reply:
The problem with that approach is that it would necessitate an additional round trip to the internet.  I don't think we want to do that.